tests/quantization/test_fp8.py

```diff
@@ -22,3 +23,37 @@ def test_load_fp16_model(vllm_runner) -> None:
         fc1 = model.model.decoder.layers[0].fc1
         assert isinstance(fc1.quant_method, Fp8LinearMethod)
         assert fc1.weight.dtype == torch.float8_e4m3fn
+
+
+@pytest.mark.skipif(
+    capability < QUANTIZATION_METHODS["fp8"].get_min_capability(),
+    reason="FP8 is not supported on this GPU type.")
+@pytest.mark.parameterize("dtype", [torch.float16, torch.bfloat16])
+def test_scaled_fp8_quant(dtype) -> None:
+
+    def fp8_quant_ref(tensor) -> torch.Tensor:
+        finfo = torch.finfo(torch.float8_e4m3fn)
+        scale = finfo.max / tensor.abs().max()
+        qweight = (tensor * scale).clamp(min=finfo.min, max=finfo.max)
+        return qweight.to(torch.float8_e4m3fn), scale
+
+    # Note that we use a shape % 4 != 0 to cover edge cases,
+    # because scaled_fp8_quant is vectorized by 4.
+    x = torch.randn(size=(10, 10), dtype=torch.float16,
```

Comment:
Just to be confident, could you use `(11, 11)`? If you look at numel 10*10 = 100 elements, which is divisible by 4

Reply:
Ah ok